core/platform/search/elastic_search_services.py

```diff
@@ -26,13 +26,16 @@
 
 import elasticsearch
 
+# This timeout is selected because 30 seconds results in reloading a
+# exploration and exp_services.load_demo() fails with a ReadTimeoutError
```

Comment:
fails --> each failing

What is the default timeout, btw?

Reply:
10

Reply:
Done